website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -84,7 +84,7 @@ We recommend using the `secondary` Button variant for primary actions and the `t
   <T.Title>Recommended button usage</T.Title>
   <T.Description>Lorem ipsum dolar sit amet, consecteu adipiscig elit null dignissim felis.</T.Description>
   <T.Button @text="Button" @color="secondary" />
-  <T.Link::Standalone @color="primary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
+  <T.Link::Standalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
```

Comment:
This example was changed to a Standalone Link, but the documentation above still advocates for the usage of a tertiary button. I think if they need to use a button (technically), they're stuck with the tertiary variant, so you may want to revert this change.

Reply:
Same as the `Alert`, I'll go ahead and change it to a button.